packages/LOLBAS/PKGBUILD

```diff
@@ -0,0 +1,24 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='LOLBAS'
+pkgver=156.f549508
+pkgrel=1
+pkgdesc='Living Off The Land Binaries And Scripts - (LOLBins and LOLScripts)'
+groups=('blackarch' 'blackarch-windows' )
+arch=('any')
+url='https://github.com/api0cradle/LOLBAS'
+depends=()
+source=('git+https://github.com/api0cradle/LOLBAS.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/$pkgname"
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd "$srcdir/$pkgname"
+  mkdir -p "$pkgdir/usr/share/windows/$pkgname/"
```

Comment:
- Insert a new line above; and
- we don't use `pkgname` variable inside the script for a few reasons. I suggest you to change all `pkgname` to `lolbas` (not capital).